frontend/src/assets/images/metadata_inline.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
This file and the image itself should be part of the backend (backend/test/assets folder?)

Reply:
Migrated to backend.